students/20th/doeunkim/feed.js

```diff
@@ -0,0 +1,295 @@
+//댓글 게시 버튼 활성화
+
+const input = document.getElementsByTagName("input")[0];
+const comment = document.getElementsByTagName("input")[1];
+const btn = document.querySelector("button");
+
+function activate() {
+    if(comment.value) {
+        btn.classList.add("active");
+        btn.removeAttribute('disabled');
+    } else {
+        btn.classList.remove("active");
+        btn.setAttribute("disabled", "disabled");
+    }
+}
+
+comment.addEventListener("input", activate);
+
+
+
+//댓글 추가&좋아요&삭제
+
+const commentForm = document.querySelector(".comments form");
+
+function handlesubmit(e) {
+    e.preventDefault();
+    const currentValue = comment.value;
+    leaveaComment(currentValue);
+    comment.value = "";
+}
+
+const inputValue = document.location.search;
+const inputId = inputValue.slice(4, inputValue.indexOf('&'));
+
+function leaveaComment(text) {
+    const newComment = document.createElement("div");
+    const contents = document.querySelector(".contents");
+    newComment.className="blahblah";
+    const id = document.createElement("p");
+    id.className = "id";
+    inputId? id.innerHTML = inputId+' ' : id.innerHTML = 'unknown ';
+    const say = document.createElement("p");
+    say.innerHTML = text;
```

Comment:
```suggestion
    const contents = document.querySelector(".contents");
    newComment.className="blahblah";
    
    const id = document.createElement("p");
    id.className = "id";
    
    inputId ? id.innerHTML = inputId + ' ' : id.innerHTML = 'unknown ';
    
    const say = document.createElement("p");
    say.innerHTML = text;
```
구분되는 뢱들은 한줄씩 띄워주시면 훨씬 읽기 좋습니다